modules/aws-outposts-requirements-limitations.adoc

```diff
@@ -8,18 +8,24 @@
 
 You can manage the resources on your Outpost similarly to those on a cloud-based AWS cluster if you configure your {product-title} cluster to accommodate the following requirements and limitations:
 
-* To extend an {product-title} cluster on AWS into an Outpost, you must have installed the cluster into an existing VPC.
+* To extend an {product-title} cluster on AWS into an Outpost, you must have installed the cluster into an existing Amazon Virtual Private Cloud (VPC).
```

Comment:
```suggestion
* To extend an {product-title} cluster on AWS into an AWS Outpost, you must have installed the cluster into an existing Amazon Virtual Private Cloud (VPC).
```

Add a link to dd.resoruces section that points to https://72291--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-vpc ?

Reply:
Best to state AWS Outpost on first occurence in a module to show owndership to AWS and thereafter just use "Outpost"?

Reply:
The VPC install is already linked above (not in the changed module). Will add "AWS" to the first instance :)